runtime/v2/shim/util_windows.go

```diff
@@ -63,3 +72,14 @@ func AnonDialer(address string, timeout time.Duration) (net.Conn, error) {
 		return c, nil
 	}
 }
+
+// AnonDialer returns a dialer for a npipe
+func AnonDialer(address string, timeout time.Duration, reconnectFlag bool) (net.Conn, error) {
```

Comment:
We should make `AnonReconnectDialer` exported. And then instead of making a `bool` for all of these just change the `loadShim` func to use `client.AnonReconnectDialer`